src/SprykerEco/Shared/Econda/EcondaConstants.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Shared\Econda;
+
+interface EcondaConstants
+{
+
+    const ECONDA_CSV_DELIMITER = "|";
```

Comment:
Those are not constants but config values (ideally in Shared\Econda\EcondaConfig class